nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/reporting/StandardEventAccess.java

```diff
@@ -338,6 +343,23 @@ ProcessGroupStatus getGroupStatus(final ProcessGroup group, final RepositoryStat
                 bytesTransferred += connectionStatusReport.getContentSizeIn() + connectionStatusReport.getContentSizeOut();
             }
 
+            if (statusAnalyticsEngine != null) {
+                StatusAnalytics statusAnalytics =  statusAnalyticsEngine.getStatusAnalytics(conn.getIdentifier());
+                if (statusAnalytics != null) {
```

Comment:
If this evaluates to `false` should we be calling `connStatus.setPredictionsAvailable(false);`? Or perhaps set it to false above and allow only this branch to override it to `true`. As-is, we explicitly set it to `false` if engine is null but leave it unset if `statusAnalytics` is null.

Reply:
@achristianson I think your recent commit resolved this, is that correct?

Reply:
I believe so, yes.